frontend/src/containers/EditMetric.css

```diff
@@ -0,0 +1,3 @@
+.width-50 {
```

Comment:
Looks like this is being defined in multiple places, can we put it in the index.css instead?